doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -538,6 +538,10 @@ Modem libraries
 
 * Updated the :ref:`nrf_modem_lib_lte_net_if` to automatically set the actual link :term:`Maximum Transmission Unit (MTU)` on the network interface when PDN connectivity is gained.
 
+* :ref:`nrf_modem_lib_readme`:
+
+  * Fixed a bug where various subsystems would be erroneusly initialized during a failed Modem library initialization.
```

Comment:
```suggestion
  * Fixed a bug where various subsystems would be erroneously initialized during a failed Modem library initialization.
```
Is the failed initialization related to any Modem library?

Reply:
Good catch.

Regarding your questions, it's related to the library initialization, so i will reword it.